tpl/setup.tpl

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Web UI config</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@1.0.2/css/bulma.min.css">
+    <style>
```

Comment:
Custom should leverage a stylesheet rather than being included with the page. You should also attempt to use [Bulma Framework](https://bulma.io/documentation/) rather than rolling your own.